app/assets/stylesheets/show.scss

```diff
@@ -0,0 +1,96 @@
+.recipe-body {
+  background-color: $white;
+  border: $black solid 2px;
+  padding: 1rem 2rem 6rem;
+
+  h2 {
+    padding-bottom: -1rem;
+  }
+
+  .creator-name {
+    color: $medium-grey;
+    margin-left: 1rem;
+    margin-top: -2.5rem;
+  }
+
+  .description {
+    margin-left: 1rem;
+  }
+
+  h5 {
+    color: $dark-grey;
+    font-family: Quattrocento;
+    font-size: 1.3rem;
+    margin: 1rem;
+  }
+
+  p {
+    margin: 1rem;
+  }
+
+  .edit-delete-recipe {
+    float: right;
```

Comment:
Address all hound violations. If you update the .yml to address them, make a reply on the hound comments